### DISCUSSION TITLE
BasicVSR++ declines after a small increase in the PSNR/SSIM of the Vid4 when trained on the Viemo 90K

### DISCUSSION THREAD

**Discussion by maodinge233** [open-mmlab/mmediting#1229] [Models]
I used the official "basicvsr_plusplus_c64n7_4x2_300k_vimeo90k_bi.py" configuration file provided by mmediting, but commented out the loading REDS model parameters and set the batch size to 6. The training results showed that the PSNR and SSIM on the The PSNR and SSIM on the Vid4 validation set rose steadily for the first 70,000 iterations, but then started to drop, as I did for two or three training attempts. I would like to know how this situation should be handled.
![image](https://user-images.githubusercontent.com/43052152/186557025-13cbc9fb-a2ed-422d-b2bd-1bd1afc94ac3.png)

**Comment by maodinge233** [comment]
batch size changed from 4x2 to 1x6

**Comment by zengyh1900** [comment]
Please check this issue. @plyfager
